Source/Urho3D/Core/Context.cpp

```diff
@@ -53,6 +67,7 @@ static void HandleIKLog(const char* msg)
 }
 #endif
 
+static Context* contextInstance_ = nullptr;

```

Comment:
I don't think that global context should go into Urho core, at least in this form. It's too restrictive now.
I think you should either move these static things into user code/Toolbox or make explicit Context::Set/GetGlobalContext to give user choice.

Reply:
Maybe while at it, [this](https://github.com/urho3d/Urho3D/blob/master/Source/Urho3D/IO/Log.cpp#L57) should be changed to make use of that global context?

Reply:
Let's fix Log it later. At least, Log is localized and optional.

I mean that if some user application use subsystems that need global context, it should set the context explicitly. Context itself shouldn't touch it.